### DISCUSSION TITLE
Uninstillation

### DISCUSSION THREAD

**Discussion by Blinding-Light** [jojo2357/Music-Discord-Rich-Presence#14] [Q&A] [answered]
I recently tried out this code and found a few things I didn't like, so I took the folder that I unzipped and deleted it, and now every time I use groove music windows yells at me. How do I stop my computer from doing this?

**Comment by jojo2357** [comment, marked as answer]
I included an "unlink with groove" shortcut that you should run. Might i ask what it is that makes you want to uninstall it and can i fix it?

**Reply by Blinding-Light** [reply]
My main problem was that I didn't like how the presence would show everyone what I'm searching on chrome for some reason.

**Reply by jojo2357** [reply]
To disable that, change the line in `DiscordRichPresence.ini` that says `chrome=true` to `chrome=false` to disable that feature
